src/content/team/Teams/teams.js

```diff
@@ -0,0 +1,13 @@
+const teams = {
```

Comment:
I can't figure out where this object is used. Is it needed?

Reply:
Yeah I was thinking about a way to store the team names as constants somewhere, but you can't import variables into JSON files. I'll delete this file. Do you have any ideas?

Reply:
I think what you have with the groupby is good